### PR TITLE
Clean up PlotId

### PR DESCRIPTION
**Pull request by Sauilitired**
Clean up plot ID usage

### CODE REVIEW

File: Bukkit/src/main/java/com/plotsquared/bukkit/entity/ReplicatingEntityWrapper.java [hunk 1]
```diff
@@ -152,16 +152,16 @@ public ReplicatingEntityWrapper(Entity entity, short depth) {
...
                 ItemFrame itemFrame = (ItemFrame) entity;
                 this.dataByte = getOrdinal(Rotation.values(), itemFrame.getRotation());
                 this.stack = itemFrame.getItem().clone();
                 return;
             case "PAINTING":
-                this.x = Math.floor(this.x);
-                this.y = Math.floor(this.y);
+                this.x = Math.floor(this.getX());
+                this.y = Math.floor(this.getY());
                 this.z = Math.floor(this.z);
```
Comment (dordsor21) [inline Bukkit/src/main/java/com/plotsquared/bukkit/entity/ReplicatingEntityWrapper.java:165]: *

Comment (Sauilitired) [reply Bukkit/src/main/java/com/plotsquared/bukkit/entity/ReplicatingEntityWrapper.java:165]: This is intentional. I cleaned up all coordinate access shit everywhere, so we use getters for everything but assignment

File: Bukkit/src/main/java/com/plotsquared/bukkit/entity/ReplicatingEntityWrapper.java [hunk 2]
```diff
@@ -407,8 +407,8 @@ private void storeLiving(LivingEntity lived) {
         this.lived.leashed = lived.isLeashed();
         if (this.lived.leashed) {
             Location location = lived.getLeashHolder().getLocation();
-            this.lived.leashX = (short) (this.x - location.getBlockX());
-            this.lived.leashY = (short) (this.y - location.getBlockY());
+            this.lived.leashX = (short) (this.getX() - location.getBlockX());
+            this.lived.leashY = (short) (this.getY() - location.getBlockY());
```
Comment (Sauilitired) [inline Bukkit/src/main/java/com/plotsquared/bukkit/entity/ReplicatingEntityWrapper.java:411]: Also intentional

Comment (dordsor21) [reply Bukkit/src/main/java/com/plotsquared/bukkit/entity/ReplicatingEntityWrapper.java:411]: you didn't clean up the z though ;)

**Pull request merged**